README.md

```diff
@@ -42,30 +42,48 @@ For more info see:
 
 ## Usage
 
-Update all plugins (must use the noop option for now):
+The repository provides several Snap plugin maintenance tools:
+
+* plugin repo sync: this updates repos with the latest license, testing, and travis ci configs
+* generate travis ci secret: create secret token for publishing binaries
+* catalog metadata: update plugin_catalog and github io page with latest github plugin metadata
+
+### Update Snap plugin repository
+
+To update all plugin repositories (must use the noop option for now):
```

Comment:
The global update is typically when we make changes to this repo, such as changing go version from 1.7.3 -> 1.7.4. I've added some common use cases on when we run global updates vs. individual updates.